.ci-mgmt.yaml

```diff
@@ -199,138 +202,3 @@ extraTests:
       matrix:
         language:
         - nodejs
-
-  provider_test:
-    if: github.event_name == 'repository_dispatch' ||
-      github.event.pull_request.head.repo.full_name == github.repository
```

Comment:
Pretty sure this condition is the reason why the test was skipped - this condition makes no sense on `master` and `release` builds.

might consider removing this instead? I understand we have sped up the tests via cache elsewhere, just a suggestion.

Reply:
Yeah it's possible to simply remove it, hm.. I was going on the assumption that reverting is the fastest way to success however it seems like we have hiccups here again.

Reply:
Yeah I'll try to go that route. It looks like we hit out of disk space. I'm pretty sure I know why - it's the excessive go dependencies of ./provider being added here.